server/server.js

```diff
@@ -14,15 +18,48 @@ const atelierHeaders = {
     'Authorization': process.env.GITHUB_TOKEN
   }
 };
+
 const s3Headers = {
   headers: {
+    'S3Bucket': process.env.S3_BUCKET,
+    'AccessKeyID': process.env.S3_ACCESS_KEY_ID,
     'Authorization': process.env.S3_TOKEN
   }
 };
 
+const s3 = new AWS.S3({
+  accessKeyId: s3Headers.headers.AccessKeyID,
+  secretAccessKey: s3Headers.headers.Authorization
+});
+
+AWS.config.update({region: 'us-east-1'});
+
+var uploadS3 = multer({
+  storage: multerS3({
+    s3: s3,
+    acl: 'public-read',
+    bucket: s3Headers.headers.S3Bucket,
+    metadata: (req, file, cb) => {
+      console.log('in multer metadata function');
+      cb(null, {fieldName: file.fieldname})
```

Comment:
Is the callback doing something specific every time? I would love a more descriptive name so we know what might be happening. If it's generic, no worries